Neumannova_Pavla_PS1.R

```diff
@@ -0,0 +1,47 @@
+# Problem 1
+# print actual time
+
+Sys.time()
+
+# create a list �bad list� with 3 positions
+
+badlist <- list(na = c(NA, NA, NA), chars = c("a", "b", "c"), bools = c(FALSE, FALSE, TRUE))
+badlist
+
+# create a data frame
+
+happyframe <- data.frame(na = c(NA, NA, NA), chars = c("a", "b", "c"), bools = c(FALSE, FALSE, TRUE))
+happyframe
+
+# Problem 2
+# lenght of numeric vector input
+x <- c(1, 2, 3, 4)
+length(x)
+
+# not using len function
+vector <- c(1,2,3,4,5,6,7,8,9)
+diff(range(vector))+1
+
+# boolean vector
+ifelse(vector %% 3 == 0, "TRUE", "FALSE")
```

Comment:
Really nice option, except for one thing:
this function checks if an element of the vector is divisible by 3 and assignes TRUE is that is the case. However, you were asked to print TRUE for every 3rd position of vector (kinda index), i.e.  assume you have vector
a = (2,6,7,8,9, 10)
then the output should be (F,F,T,F,F,T)

I understand my formulation might have been unclear and won't subtract any points for that. Otherwise, very neat solution.